pkg/endpoint/bpf.go

```diff
@@ -727,7 +727,16 @@ func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenC
 			Debug("BPF header file unchanged, skipping BPF compilation and installation")
 	}
 
+	// Hook the endpoint into the endpoint table and expose it
+	stats.mapSync.Start()
+	err = lxcmap.WriteEndpoint(epInfoCache)
```

Comment:
This will introduce drops now as the policy map is not ready when the endpoint is announced.

Reply:
It is. There are 2 policy map updates. The 1st (and most important one) is done on line 607. The only purpose of the 2nd policy map update below is to add newly created L7 redirects.

Reply:
OK, it still moves the lxcmap update in front of waiting for the ctmap cleanup to finish so we could clean up while initial active connections can start being created.

Reply:
We may actually need to wait for the ct cleanup to complete before the BPF generation is performed

Reply:
OK, I will add a commit to do that.
I will keep the setting of `Endpoint.ctCleaned` to `true` where it is, i.e. after waiting for proxy ACK, because I don't want to acquire the lock earlier.